cookbook/langsmith_cookbook/langsmith_stream.py

```diff
@@ -0,0 +1,80 @@
+"""LangSmith + Mirascope example of streaming OpenAI chat with history."""
+import os
+
+from langsmith import wrappers
+from langsmith_config import Settings
+
+from mirascope import OpenAICallParams, OpenAIChat, Prompt
+from mirascope.prompts.messages import Message
+
+settings = Settings()
+
+os.environ["LANGCHAIN_API_KEY"] = settings.langchain_api_key
+os.environ["LANGCHAIN_TRACING_V2"] = settings.langchain_tracing_v2
+os.environ["OPENAI_API_KEY"] = settings.openai_api_key
+
+
+class ChatPrompt(Prompt):
```

Comment:
can you also please add another non-langchain example in the examples folder of adding history? like chat_history.py or something?

Reply:
so you want me to split this into two?

Reply:
no a copy-cat example `examples/chat_history.py` that shows how to inject history without INJECT

Reply:
ooh gotcha